poly-commitment/src/evaluation_proof.rs

```diff
@@ -119,7 +119,11 @@ pub fn combine_polys<G: CommitmentCurve, D: EvaluationDomain<G::ScalarField>>(
     if !plnm_evals_part.is_empty() {
         let n = plnm_evals_part.len();
         let max_poly_size = srs_length;
-        let num_chunks = n / max_poly_size;
+        let num_chunks = if n % max_poly_size == 0 {
```

Comment:
Maybe `n.div_ceil(max_poly_size)` is equivalent?

Reply:
Nevermind, this API is still unstable ...

Reply:
Ah, stable for unsigned integers though!

Reply:
Nice. Didn't know. Updating.

Reply:
Yes, unstable unfortunately. Not working for usize :(. Good to remember though.